seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-druid/src/main/java/org/apache/seatunnel/flink/druid/source/DruidSource.java

```diff
@@ -169,6 +179,8 @@ private RowTypeInfo getRowTypeInfo(String jdbcURL, String datasource, Collection
             names[i] = field;
             i++;
         }
+        long count = Arrays.stream(typeInformation).filter(t -> null == t).count();
+        System.out.println("typeInformation null:::" + count);
```

Comment:
remove?

Reply:
done

Reply:
I

> 

I have commit new commits, should I push a new pr?

Reply:
> 

I suggest you don't push a new pr if the new pr and this pr have many code dependency. Of course, if there is no code dependency between two prs, you can submit.